modules/extensions-spring-support/src/test/resources/schemas/vegan.xsd

```diff
@@ -82,18 +82,6 @@
                                 </xs:choice>
                             </xs:complexType>
                         </xs:element>
-                        <xs:element minOccurs="0" maxOccurs="1" name="parameterized-functions">
```

Comment:
I doubt this is OK, since the definition for this is: 
@Parameter
@Optional
private Function<Event, Map<String, String>> parameterizedFunction;

If we are making the Functions "Expression.REQUIRED" or if we don't care at all because the Functions will be removed, we can accept this. Other way, we have to fix this side effect